src/acl/external/AD_group/Makefile.am

```diff
@@ -19,6 +19,7 @@ ext_ad_group_acl_SOURCES = ext_ad_group_acl.cc
 # TODO: test for these libraries in required.m4
```

Comment:
If this PR make "This helper can be compiled only using Microsoft Visual Studio." statement two lines above false, please adjust that comment accordingly.

Reply:
Fixed; I can confirm that this builds in mingw (using docker image squidcache/buildfarm-fedora-mingw:stable)